client/knowledge_base_search/src/containers/QuestionBarContainer/styles.scss

```diff
@@ -10,14 +10,21 @@
     display: flex;
   }
 
+  &--bottom {
+    margin-top: 0.5rem;
+    display: flex;
+    float: right;
```

Comment:
floats are the old way to do what flexbox does, so you probably shouldn't have this anywhere when using flexbox. instead look up how to align the content https://css-tricks.com/snippets/css/a-guide-to-flexbox/#article-header-id-6